test/unit/backend/server.spec.js

```diff
@@ -27,4 +27,16 @@ describe('Server', () => {
         });
     });
   });
+
+  describe('/api/movies', () => {
+    it('should GET all the movies', (done) => {
+      chai.request(server)
+        .get('/api/movies')
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.body.should.be.a('array');
+          done();
+        });
+    });
+  });
 });
```

Comment:
Ah, I see! Should we wait until that's sorted or we going ahead with this and keep it in the backlog?